modules/apps/staging/staging-impl/src/main/java/com/liferay/staging/internal/StagingGroupHelperImpl.java

```diff
@@ -146,6 +146,11 @@ public Group fetchRemoteLiveGroup(long groupId) {
 		return fetchRemoteLiveGroup(_groupLocalService.fetchGroup(groupId));
 	}
 
+	@Override
+	public String getCompanyGroupFriendlyURL() {
+		return CompanyGroupConstants.FRIENDLY_URL;
```

Comment:
Maybe we should move this constract to an API module. I am not sure if it makes to create a method just to return always the same constant value